backend/volumes.py

```diff
@@ -213,17 +214,43 @@ def edit(self, edits: dict) -> dict:
 		"""
 		logging.debug(f'Editing volume {self.id}: {edits}')
 		monitored = edits.get('monitor')
-		if monitored == True:
+		if monitored:
 			self._monitor()
-		elif monitored == False:
+		else:
 			self._unmonitor()
```

Comment:
With your changes, if the value of `monitor` evaluates to `False` _in any way_ (e.g. `0` or `""`), the volume will be unmonitored. This is, in my opinion, not the desired behaviour. The user of the API should explicitly supply a boolean value for changes to be made. Any other value, like `0` (false), `""` (false) or `"foobar"` (true), should not be allowed as a way to set the monitored status. And with your changes, they will.

Please remove these changes.

Reply:
Proper global input validation is indeed something that is needed as this project scales. But for now the current implementation works.